src/modules/search-time/types.ts

```diff
@@ -0,0 +1,11 @@
+export const SEARCH_MODES = ['now', 'arriveBy', 'departBy'] as const;
+export type SearchMode = (typeof SEARCH_MODES)[number];
+
+export type SearchTime =
+  | {
+      mode: 'now';
+    }
+  | {
+      mode: 'arriveBy' | 'departBy';
+      dateTime: number;
```

Comment:
Should this be Date?

Reply:
It certainly can. Will look into why it was a number and possibly make it a Date.

Reply:
This was the main reason IIRC. 
<img width="663" alt="2023-11-08 at 20 29 20@2x" src="https://github.com/AtB-AS/planner-web/assets/14045515/2ed878c3-6d13-4e24-9502-1f154b3de4b7">

Need to handle the conversion to date somewhere, and if it can't be in the `getServerSideProps` I'm afraid it can create more noise than just using the millisecond representation.

I might be thinking wrong here and there's an easy solution to this problem.

Reply:
Aah, thats right. This is passed over server side props. https://github.com/blitz-js/next-superjson-plugin is a solution for this, but I think number is just fine for now 😅

Reply:
TIL about that plugin 🤩